SETUP.md

```diff
@@ -15,6 +15,16 @@ stack test
 #### If you get an error message like this...
 
 ```
+No .cabal file found in directory
+```
+
+You are probably running an old stack versions and need
+to upgrade it.
+
+#### Otherwise, if you get an error message like this...
+
+
```

Comment:
probably just one newline instead of two, unless you reaaaally want two for some reason